go.mod

```diff
@@ -1,48 +1,48 @@
 module github.com/grafana/grafana-bigquery-datasource
 
-go 1.22
+go 1.22.7
 
-toolchain go1.22.1
+toolchain go1.23.4
 
 require (
 	cloud.google.com/go v0.112.1
 	cloud.google.com/go/bigquery v1.59.1
-	github.com/grafana/grafana-google-sdk-go v0.1.0
-	github.com/grafana/grafana-plugin-sdk-go v0.258.0
+	github.com/grafana/grafana-google-sdk-go v0.2.1
```

Comment:
I bumped this as well. But this is not adding any code changes just dependency bumps.